sdf-schema.json

```diff
@@ -122,104 +127,138 @@
           },
           "minitems": 1
         },
-        "optional": {
-          "type": "boolean"
+        "required": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "minitems": 1
         }
-      }
+      },
+      "required": ["id"]
```

Comment:
is this correct?
e.g. where is id listed/defined?

Reply:
This is to make the ID quality mandatory in all definitions. It refers to the "id" definition in the list of common qualities, i.e. at the same level in the file as the "properties" keyword for the elements we want to declare as required. The syntax is correct.